mobile/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
Necessary for FCM connection. Safe to commit to VC according to Google and Stackoverflow